storm-server/src/test/java/org/apache/storm/scheduler/resource/strategies/priority/TestFIFOSchedulingPriorityStrategy.java

```diff
@@ -43,54 +43,52 @@
 
     @Test
     public void testFIFOEvictionStrategy() {
-        try (Time.SimulatedTime sim = new Time.SimulatedTime()) {
```

Comment:
We might not want to delete this because it simulates the Time and saves time spending on unit test on some cases.

Reply:
Thanks for catching. I will revert this and investigate further.